.buildkite/aggregate.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+  - label: "Aggregate"
+    env:
+      # This is a child pipeline, so we MUST put this here to inherit privileges from the parent pipeline.
+      BUILDKITE_PLUGIN_CRYPTIC_BASE64_SIGNED_JOB_ID_SECRET: ${BUILDKITE_PLUGIN_CRYPTIC_BASE64_SIGNED_JOB_ID_SECRET?}
+
+    plugins:
+      - staticfloat/cryptic#v2:
+          variables:
+            - DOCUMENTER_KEY="U2FsdGVkX1+JU3NwkwSLBFtnGQnPOpZmfZx9TaomLdMtRqB2YHncIzZ2oIm9F7s8HMd6OWWYXBsCmbodqbWRpsmgPmB+jXqRTd5xaI9y8Dtq4grLt0EIHiIeTXQxzTSS8MRYiUhvL9cjPr9IgJh4I1dzPcB9cS+dKwxAXkFfUeam9QAFQnG9kvrMeJdfcYc4EKX5KsGJfAFM/YRKtEWp6yndWcP3fl6UR6NWWQrCiwaJFjZ+TMNDat5cJTcYklD24G4soLEtZRdYU0Uq066e1OBsQMjRbXCfEyifUupTdp/gRsnHMU31BVO9ISSkTf2eOxhtIeTgY7Wde1PrGxk8QH8UCoXiZpbkacOszF11BbOAdQRpqhP0lbiaPyNeBwREK/8sz/K1BBT5+atyGWml8Km/VxSKJV03OLoJrszgP8c9WKi+jxhsn0EZ3PRRlmjLwWjrTfrh9UbJmgZ2ViYeP9VVwPK+RolT11BmyUQUfY+ajT9fUOvyxKytLs13S/qgWHyW5X66dcw5htVrp21vbfTtKoeZALueV2MfLXbBBveZKnrsz3K4HVBDYNyB43+/2MBCWmSIbSYQPj0jpF2tggo0X7fGjKctW+op8CvuNmqM9SBWL0biWG7sINcC6TSW1JnH77yA0nOPhiavBI38p2yC7q4+oCu3njKoxOJO3YWCie3uU0lwHNVyfCsajLIk/kIQwAAw1IS4y0F180g6Q5+uhXVdT2VhDnrUNVmfdkIAFXy2nSB8RE2dHqKw04kiANW9vIa/1jAhkG8BnRAKiA=="
+      - staticfloat/ssh-agent:
+          keyvars:
+            - "DOCUMENTER_KEY"
+      - JuliaCI/julia#v1:
+          version: "1.8"
+
+    command: |
+      # Install dependencies
+      julia --project=docs/ -e 'using Pkg; Pkg.develop(PackageSpec(path=pwd())); Pkg.instantiate()'
+      # Aggregate and Deploy
+      git config user.name buildkite
+      git config user.email julia@buildkite.com
+      julia --project=docs/ docs/make_aggregate.jl
+
+    agents:
+      queue: "juliaecosystem"
+      sandbox.jl: true
```

Comment:
Answering a question on slack: yes, I think we probably only want to do this on `x86_64`

Reply:
Resolved in 0bc3213.